bigchaindb/commands/utils.py

```diff
@@ -95,6 +96,34 @@ def start(parser, argv, scope):
     return func(args)
 
 
+def mongodb_host(host):
```

Comment:
Not sure where to put this. This is basically a special type validation method for `ArgumentParser`